test/e2e-techpreview/onclusterlayering_test.go

```diff
@@ -858,16 +858,28 @@ func assertBuildPodIsAsExpected(t *testing.T, cs *framework.ClientSet, mosb *mcf
 	mcoImages, err := ctrlcommon.GetImagesConfig(context.TODO(), cs.GetKubeclient())
 	require.NoError(t, err)
 
-	buildPod, err := cs.CoreV1Interface.Pods(ctrlcommon.MCONamespace).Get(context.TODO(), mosb.Status.BuilderReference.PodImageBuilder.Name, metav1.GetOptions{})
+	buildJob, err := cs.BatchV1Interface.Jobs(ctrlcommon.MCONamespace).Get(context.TODO(), mosb.Status.BuilderReference.PodImageBuilder.Name, metav1.GetOptions{})
 	require.NoError(t, err)
 
+	buildPods, err := cs.CoreV1Interface.Pods(ctrlcommon.MCONamespace).List(context.TODO(), metav1.ListOptions{LabelSelector: fmt.Sprintf("job-name=%s", buildJob.Name)})
```

Comment:
question: Should this label be `batch.kubernetes.io/job-name` instead of just `job-name`? I ask because of this: https://kubernetes.io/docs/concepts/workloads/controllers/job/#job-labels

Reply:
`job-name` is a label as well, but I switched it to use the one from the k8s docs.

Reply:
I changed this back to `job-name` because for some reason when I was using `batch.kubernetes.io/job-name` it wasn't being able to find the pods for the job. Kept getting a nil pod causing the tests to fail.

When doing this manually on an OCP cluster, I see both labels applied to the pod so having the tests use `job-name` shouldn't be an issue.